Ga 2011 01

sarang-amondikar opened the pull request:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] Please ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

sarang-amondikar committed:
changes for 2020-11-01 GA

sarang-amondikar committed:
re-formatting with Beautify

sarang-amondikar committed:
re-formatting using Beautify

wilcobmsft commented on specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json:
```diff
@@ -18,774 +18,827 @@
+                    "enum": [
+                        "Unknown",
+                        "Active",
+                        "Deleted",
```
I thought we were going to remove "Deleted" and merge PendingReview/UnderReview.

sarang-amondikar replied:
Correct for Deleted, I will remove it, for merging of two states PendingReview/UnderReview, still waiting on response from Trade team

sarang-amondikar replied:
removed Deleted

wilcobmsft commented on specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json:
```diff
@@ -18,774 +18,827 @@
+                    "readOnly": true
+                },
+                "endDate": {
+                    "description": "The date when this benefit ends.",
```
nit: A spending limit isn't really a benefit?

wilcobmsft commented on specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json:
```diff
@@ -18,774 +18,827 @@
+            }
+        },
+        "spendingLimitDetails": {
+            "description": "The details of spending limit policy set on billingProfile.",
```
"...set on a billing profile."

sarang-amondikar replied:
updated

wilcobmsft commented on specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingAccount.json:
```diff
@@ -18,774 +18,827 @@
+                        "Active",
+                        "Expired",
+                        "LimitReached",
+                        "LimitRemoved"
```
Should it just be "Removed"?

sarang-amondikar replied:
updated

sarang-amondikar committed:
Nit: updated description for SpendingLimitDetails.

updated SpendingLimitStatus enum
removed deleted account status as deleted account is never returned

sarang-amondikar committed:
nit

jbuls committed:
Added invoiceRecipients to billingProfile

jbuls committed:
Merge pull request #5 from sarang-amondikar/jbuls/invoiceRecipients/02

Added invoiceRecipients to billingProfile

sarang-amondikar commented on specification/billing/resource-manager/Microsoft.Billing/stable/2020-11-01/billingProfile.json:
```diff
@@ -176,6 +176,13 @@
           "type": "integer",
           "readOnly": true
         },
+        "invoiceRecipients": {
```
how about invoiceRecipientsEmail?

udmalik committed:
Adding account status reason code

udmalik committed:
Merge pull request #6 from sarang-amondikar/AddAccountStatusReasonCode

Adding account status reason code.

Commit:
Introducing billingRoleAssignment API in new GA version

Introducing billingRoleAssignment API in new GA version

Commit:
Formatted the files

asarkar84 committed:
Merge pull request #7 from sarang-amondikar/ansar/addBillingRoleAssignment

Introducing billingRoleAssignment API in new GA version

sarang-amondikar committed:
Add parameters

sarang-amondikar committed:
re-introduce deleted account status and removed PendingReview as only UnderReview will be used for TradeScreening
